pyfiglet/fonts/ansi-shadow.flf

```diff
@@ -0,0 +1,722 @@
+flf2a$ 7 7 13 0 7 0 64 0
+Font Author: ?
+
+More Info:
+
+https://web.archive.org/web/20120819044459/http://www.roysac.com/thedrawfonts-tdf.asp
+
+FIGFont created with: http://patorjk.com/figfont-editor
```

Comment:
What's the license story behind this font? Where did it come from, and what is its license? See also #59. I'm reluctant to accept a PR which is not clear on this front.

Reply:
Yeah - based on the analysis we've just gone through, this would be another "contrib" font at best.  Possibly a better way to handle these sorts of requests is to add URLs to common locations for font definitions (rather than the fonts themselves)?